build.sbt

```diff
@@ -38,6 +39,12 @@ def commonSettings(subProject: Option[String]): Seq[Setting[_]] = {
         Set(organization.value %% artifact.value % mimaPreviousVersion.value)
     },
 
+    mimaBinaryIssueFilters ++= Seq(
+      ProblemFilters.exclude[ReversedMissingMethodProblem](
+        "org.scalacheck.ops.time.ImplicitJavaTimeGenerators.arbOffsetDateTime"
+      )
```

Comment:
Why is this needed?

Reply:
Not sure if this is relevant but the only reason I added this filter is because `mimaReportBinaryIssues` was being executed in the old travis Ci which caused the CI to fail. The newly generated github actions doesn't use `mimaReportBinaryIssues` so I can also remove this filter if you desire.